resources/views/admin/basis/css.blade.php

```diff
@@ -4,3 +4,4 @@
 <link href="/libs/adminLTE/dist/css/AdminLTE.min.css" rel="stylesheet" type='text/css'>
 <link href="/libs/adminLTE/dist/css/skins/_all-skins.min.css" rel="stylesheet" type="text/css">
 <link href='/css/main.css' rel='stylesheet' type='text/css'>
+<link href='/css/settings.css' rel='stylesheet' type='text/css'>
```

Comment:
This is specific CSS for settings. So please put it in child view in section `css`.